packages/nimble-components/src/unit/volt/models/volt-unit-scale-formatter.ts

```diff
@@ -0,0 +1,25 @@
+import { ManuallyTranslatedUnitScaleFormatter } from '../../base/models/manually-translated-unit-scale-formatter';
+import { UnitPrefix, metricPrefixes } from '../../base/models/unit-prefix';
+import { UnitTranslation } from '../../base/models/unit-translation';
+
+/**
+ * Formatter for numbers with voltage units
+ */
+export class VoltUnitScaleFormatter extends ManuallyTranslatedUnitScaleFormatter {
+    protected override getUnitTranslations(): Map<string, UnitTranslation> {
+        const unitTranslations = new Map<string, UnitTranslation>();
```

Comment:
We can make this map once since it's all constants. Can use a static initialization block to set it up: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Static_initialization_blocks

Or even more direct use the array syntax to initialize with a single constant: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/Map#creating_a_new_map